changes_test.go

```diff
@@ -3328,7 +3346,7 @@ func (s *changesSuite) TestSimpleBundleEmptyModel(c *gc.C) {
 	expectedChanges := []string{
 		"upload charm cs:django-4",
 		"deploy application django using cs:django-4",
-		"expose django",
+		"expose endpoint(s) www of django to space(s) dmz and CIDR(s) 13.37.0.0/16",
```

Comment:
Just for completeness, I wouldn't mind some endpoint/space/CIDR length > 1 tests.

Reply:
Sure, let me add one